src/popover/popover.directive.ts

```diff
@@ -54,10 +66,11 @@ export class PopoverDirective implements OnInit, OnDestroy {
   @Output() public onHidden: EventEmitter<any>;
 
   private _popover: ComponentLoader<PopoverContainerComponent>;
+  private _outsideClickListener: Function;
 
-  public constructor(_elementRef: ElementRef,
-                     _renderer: Renderer,
-                     _viewContainerRef: ViewContainerRef,
+  public constructor(protected _elementRef: ElementRef,
```

Comment:
Works with the other approach too
![image](https://cloud.githubusercontent.com/assets/8938562/22279784/1c7a9716-e2cd-11e6-89fa-63b09e724c86.png)